sources/HUBViewControllerImplementation.m

```diff
@@ -73,7 +73,7 @@ @interface HUBViewControllerImplementation () <HUBViewModelLoaderDelegate, HUBIm
 @property (nonatomic, strong, readonly) HUBComponentUIStateManager *componentUIStateManager;
 @property (nonatomic, strong, readonly) HUBComponentReusePool *childComponentReusePool;
 @property (nonatomic, strong, nullable) id<HUBViewModel> viewModel;
-@property (nonatomic, strong, nullable) HUBViewModelDiff *lastViewModelDiff;
+@property (nonatomic, strong, nullable) NSArray<HUBViewModelDiff *> *pendingDiffs;
```

Comment:
Also, looking at the rest of the code, doesn't it make more sense for it to be a `NSMutableArray`?